### DISCUSSION TITLE
使用 hanwckf/bl-mt798x 引导主线 OpenWrt 固件

### DISCUSSION THREAD

**Discussion by 1715173329** [1715173329/blog#19] [Comments]
https://blog.imouto.in/post/openwrt/2024/use-bl-mt798x-for-mainline/

**Comment by 471309682** [comment]
https://drive.wrt.moe/uboot/挂了打不开，想要支持主线openwrt的JCGQ30PROuboot

**Comment by Sumire-Heanna** [comment]
Squashfs itb failed when flashing, same outcome as flashing bl2. Any modification required to take in the mainstream OpenWrt files?

**Comment by vincascm** [comment]
JCG Q30 Pro , 已刷入[immortalwrt](https://github.com/immortalwrt/immortalwrt) 23.05 custom U-Boot layout 的固件，现在想刷入 immortalwrt 24.10 或 OpenWrt 24.10，有没有更便捷安全的方式，上述的操作步骤，其实对于普通用户来说，需要的的上下文和前置知识仍然比较多，而且操作本身也有风险。
非常感谢

**Comment by kousyougi** [comment]
> JCG Q30 Pro , 已刷入[immortalwrt](https://github.com/immortalwrt/immortalwrt) 23.05 custom U-Boot layout 的固件，现在想刷入 immortalwrt 24.10 或 OpenWrt 24.10，有没有更便捷安全的方式，上述的操作步骤，其实对于普通用户来说，需要的的上下文和前置知识仍然比较多，而且操作本身也有风险。 非常感谢

https://www.vinoca.org/jcg-q30-procong-23-05sheng-ji-dao-24-10/

搜到這篇。有人成功了。但過程沒看懂。

**Reply by 1715173329** [reply]
你所提到的这篇文章使用的依然是 openwrt 官方的 uboot。

我个人是不推荐使用如本文提到的这类第三方 uboot，因为它不受官方支持。
在向 openwrt 官方提交设备支持时，我们一般都会附上对应的刷入方法，照着那个来就行了。
